hr_timesheet_no_closed_project_task/views/hr_timesheet_sheet_view.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record model="ir.ui.view" id="hr_timesheet_sheet_form">
+            <field name="name">hr.timesheet.sheet.form (hr_timesheet_no_closed_project_task)</field>
+            <field name="model">hr_timesheet_sheet.sheet</field>
+            <field name="inherit_id" ref="hr_timesheet_task.hr_timesheet_sheet_form_with_activity"/>
+            <field name="arch" type="xml">
+                <xpath
+                    expr="//field[@name='timesheet_ids']/tree[@string='Timesheet Activities']/field[@name='account_id']"
+                    position="attributes">
+                    <attribute name="domain">[('type','in',['normal','contract']),('state', '&lt;&gt;', 'close'),('use_timesheets','=',1),'|', ('project_ids', '=', False), ('project_ids.state', 'not in', ('close', 'cancelled'))]</attribute>
```

Comment:
This line is a bit long (240 chars). Can you seperate the domain expression in multiple lines ?